src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -343,12 +344,7 @@ protected virtual Task<bool> HandleSignOutCallbackAsync()
             };
 
             await Events.RedirectToIdentityProvider(redirectContext);
-            if (redirectContext.HandledResponse)
-            {
-                Logger.RedirectToIdentityProviderHandledResponse();
-                return;
-            }
-            else if (redirectContext.Skipped)
+            if (redirectContext.Handled)
             {
                 Logger.RedirectToIdentityProviderSkipped();
```

Comment:
Heh so I removed Handled, and kept skip, then renamed Skip back to Handled...  nuked the log extensions for Skip